modules/signals/src/signal-store-models.ts

```diff
@@ -11,9 +11,9 @@ export type StateSignals<State> = IsKnownRecord<Prettify<State>> extends true
     }
   : {};
 
-export type SignalsDictionary = Record<string, Signal<unknown>>;
+export type SignalsDictionary = Record<string | symbol, Signal<unknown>>;
```

Comment:
@markostanimirovic @timdeschryver: This could be seen as a breaking change.

If I leave it to string only (`SignalsDictionary` & `MethodsDictionary`), tests and build would also be successful.

What's your take on this?

Reply:
@timdeschryver honestly, I have no idea. I just saw that this is a change to public type, and we don't know if there is some code out there that depends, that it is only a string and nothing else...

Reply:
Since `Record<string, ...>` already supports symbols as keys, we can avoid changing `SignalsDictionary` and `MethodsDictionary` types just in case.

Reply:
Done